onvm/onvm_mgr/onvm_nf.c

```diff
@@ -247,6 +253,12 @@ onvm_nf_stop(struct onvm_nf_info *nf_info) {
         service_id = nf_info->service_id;
         nf_status = nf_info->status;
 
+        /* Cleanup the allocated tag */
+        if (nf_info->tag) {
```

Comment:
Not sure why this is an addition its already in the dev branch. Can you pull upstream dev?